responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -206,20 +207,24 @@ def load_config(config_directory_path):
 
         return counterfactual_config
 
-    def get_result(self):
+    def get_result(self, include_counterfactual_obj=True):
         """Returns the dictionary representation of result of the computation
            in the CounterfactualConfig.
 
+        :param include_counterfactual_obj: Whether to include serialized
+                                           version of counterfactual object.
+        :type method: bool
         :return: The dictionary representation of result in the
                  CounterfactualConfig.
         :rtype: dict
         """
         result = {}
-        if self.counterfactual_obj is not None:
-            result[CounterfactualConfig.COUNTERFACTUAL_OBJ] = \
-                self.counterfactual_obj.to_json()
-        else:
-            result[CounterfactualConfig.COUNTERFACTUAL_OBJ] = None
+        if include_counterfactual_obj:
+            if self.counterfactual_obj is not None:
+                result[CounterfactualConfig.COUNTERFACTUAL_OBJ] = \
```

Comment:
minor nit: I think you can avoid several slashes by making CounterfactualConfig.COUNTERFACTUAL_OBJ a constant at the top, as well as some other constants referenced here.
Something like:
```
COUNTERFACTUAL_OBJ = CounterfactualConfig.COUNTERFACTUAL_OBJ
HAS_COMPUTATION_FAILED = CounterfactualConfig.HAS_COMPUTATION_FAILED
...
```

Reply:
will probably keep it thi way.